packages/api/src/client/types/com/atproto/repo/putRecord.ts

```diff
@@ -5,38 +5,25 @@ import { Headers, XRPCError } from '@atproto/xrpc'
 
 export interface QueryParams {}
 
-export interface CallOptions {
-  headers?: Headers;
-  qp?: QueryParams;
-  encoding: 'application/json';
-}
-
 export interface InputSchema {
-  /**
-   * The DID of the repo.
-   */
```

Comment:
Not a big deal, but I was a fan of this documentation appearing inline during development :) We can always add it back later if we'd like—happy to peel off an issue if it's of interest.

Reply:
I'll add that to the PR tasks, probably not too hard to do in the code generation.